apps/vctags/compile.sh

```diff
@@ -0,0 +1 @@
es.v3 ../../aeneas/src/mach/MachStackifier.v3 ../../aeneas/src/mach/MachType.v3 ../../aeneas/src/mach/Memory.v3 ../../aeneas/src/mach/MoveResolver.v3 ../../aeneas/src/mach/RegAlloc.v3 ../../aeneas/src/mach/ShadowStackSpiller.v3 ../../aeneas/src/main/Aeneas.v3 ../../aeneas/src/main/Compiler.v3 ../../aeneas/src/main/Debug.v3 ../../aeneas/src/main/Error.v3 ../../aeneas/src/main/Option.v3 ../../aeneas/src/main/Version.v3 ../../aeneas/src/mips/MipsAssembler.v3 ../../aeneas/src/ssa/Gvn.v3 ../../aeneas/src/ssa/SsaBlockOrder.v3 ../../aeneas/src/ssa/SsaBuilder.v3 ../../aeneas/src/ssa/SsaContext.v3 ../../aeneas/src/ssa/SsaDeconstructor.v3 ../../aeneas/src/ssa/SsaGraph.v3 ../../aeneas/src/ssa/SsaIcGen.v3 ../../aeneas/src/ssa/SsaInliner.v3 ../../aeneas/src/ssa/SsaMap.v3 ../../aeneas/src/ssa/SsaOptimizer.v3 ../../aeneas/src/ssa/SsaPrinter.v3 ../../aeneas/src/ssa/SsaRebuilder.v3 ../../aeneas/src/ssa/SsaStructurer.v3 ../../aeneas/src/ssa/SsaTarget.v3 ../../aeneas/src/ssa/Ssa.v3 ../../aeneas/src/ssa/SsaVerifier.v3 ../../aeneas/src/ssa/VstSsaGen.v3 ../../aeneas/src/test/Regression.v3 ../../aeneas/src/types/Bool.v3 ../../aeneas/src/types/Byte.v3 ../../aeneas/src/types/Float.v3 ../../aeneas/src/types/Function.v3 ../../aeneas/src/types/Int.v3 ../../aeneas/src/types/Tuple.v3 ../../aeneas/src/types/Type.v3 ../../aeneas/src/types/Void.v3 ../../aeneas/src/util/Arrays.v3 ../../aeneas/src/util/BitMatrix.v3 ../../aeneas/src/util/Char.v3 ../../aeneas/src/util/DataBuffer.v3 ../../aeneas/src/util/Edge.v3 ../../aeneas/src/util/IO.v3 ../../aeneas/src/util/Label.v3 ../../aeneas/src/util/List.v3 ../../aeneas/src/util/Map.v3 ../../aeneas/src/util/Poly.v3 ../../aeneas/src/util/Progress.v3 ../../aeneas/src/util/Strings.v3 ../../aeneas/src/util/Terminal.v3 ../../aeneas/src/util/UID.v3 ../../aeneas/src/util/Vector.v3 ../../aeneas/src/util/WorkQueue.v3 ../../aeneas/src/v3/Mangle.v3 ../../aeneas/src/v3/SystemModule.v3 ../../aeneas/src/v3/TypeSystem.v3 ../../aeneas/src/v3/V3Array.v3 ../../aeneas/src/v3/V3Class.v3 ../../aeneas/src/v3/V3Component.v3 ../../aeneas/src/v3/V3Enum.v3 ../../aeneas/src/v3/V3Op.v3 ../../aeneas/src/v3/V3.v3 ../../aeneas/src/vst/FpSketch.v3 ../../aeneas/src/vst/NumberParser.v3 ../../aeneas/src/vst/ParserState.v3 ../../aeneas/src/vst/Parser.v3 ../../aeneas/src/vst/Token.v3 ../../aeneas/src/vst/VarEnvironment.v3 ../../aeneas/src/vst/Verifier.v3 ../../aeneas/src/vst/VstMatcher.v3 ../../aeneas/src/vst/VstPrinter.v3 ../../aeneas/src/vst/Vst.v3 ../../aeneas/src/wasm/CfgRestructurer.v3 ../../aeneas/src/wasm/WasmCodeGen.v3 ../../aeneas/src/wasm/WasmOp.v3 ../../aeneas/src/wasm/WasmTarget.v3 ../../aeneas/src/x86/DarwinModule.v3 ../../aeneas/src/x86/LinuxModule.v3 ../../aeneas/src/x86/X86Assembler.v3 ../../aeneas/src/x86/X86Backend.v3 ../../aeneas/src/x86/X86CodeGen.v3 ../../aeneas/src/x86/X86Darwin.v3 ../../aeneas/src/x86/X86Linux.v3 ../../aeneas/src/x86/X86MacroAssembler.v3 ../../aeneas/src/x86/X86RegSet.v3 ../../aeneas/src/x86/X86Runtime.v3 ../../aeneas/src/x86/X86Target.v3
```

Comment:
Instead of compile.sh, please add a DEPS file that lists the Virgil files this app depends on.
AFAICT that should be just one line:
../../aeneas/src/*/*.v3

Reply:
Done